examples/favorites_advanced/getx_favorites_advanced/lib/feature_puppy/controllers/puppy_extra_details_controller.dart

```diff
@@ -0,0 +1,49 @@
+import 'dart:async';
+// import 'package:flutter/cupertino.dart';
+import 'package:getx_favorites_advanced/feature_puppy/controllers/puppy_list_controller.dart';
+import 'package:get/get.dart';
+
+import 'package:favorites_advanced_base/models.dart';
+import 'package:favorites_advanced_base/repositories.dart';
+
+class PuppyExtraDetailsController extends GetxController {
+  final _repository = Get.find<PuppiesRepository>();
```

Comment:
@RomanovaPrime what about introducing DI here? Thus, writing test will be easier as you will be able to inject a mock repository ;)

Reply:
Corrected